kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/KafkaTopicManager.java

```diff
@@ -226,26 +218,14 @@ private void lookupBroker(String topicName,
                     retFuture.complete(pair.getLeft());
                 })
                 .exceptionally(th -> {
-                    long waitTimeMs = backoff.next();
-
-                    if (backoff.isMandatoryStopMade()) {
-                        log.warn("[{}] getBroker for topic {} failed, retried too many times {}, return null."
-                                        + " throwable: ", requestHandler.ctx.channel(), topicName, waitTimeMs, th);
-                        retFuture.complete(null);
-                    } else {
-                        log.warn("[{}] getBroker for topic failed, will retry in {} ms. throwable: ",
-                            topicName, waitTimeMs, th);
-                        requestHandler.getPulsarService().getExecutor()
-                            .schedule(() -> lookupBroker(topicName, backoff, retFuture),
-                                waitTimeMs,
-                                TimeUnit.MILLISECONDS);
```

Comment:
How can you ensure that it will look up next time if these code was removed? IMO, If lookup failed, only a warn log would be printed and the `retFuture` would never be completed.

Reply:
1, if `retFuture` can not get the topic address, the client will get an error and retry.
2,  I tested that if `retFuture`  can not be completed, `whenComplete` will not execute. 
if `retFuture`  be completed with `null`,  `LOOK_CACHE` will cache a `retFuture`  with `null` value. here we can add `retFuture.complete(null);` back, because `LOOK_CACHE` remove this topic cache if the address is `null`.

Reply:
the latest commit only removes the schedule thread for lookup.